net/src/Substrate.Gear.Client/Model/Types/Primitive/NonZeroU64.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Substrate.NetApi.Attributes;
+using Substrate.NetApi.Model.Types.Base;
+using Substrate.NetApi.Model.Types.Metadata.Base;
+using Substrate.NetApi.Model.Types.Primitive;
+
+namespace Substrate.Gear.Client.Model.Types.Primitive;
+
+/// <summary>
+/// NonZeroU64
+/// </summary>
+[SubstrateNodeType(TypeDefEnum.Composite)]
+public sealed class NonZeroU64 : BaseType
+{
+
+    /// <summary>
+    /// >> value
+    /// </summary>
+    public required U8 Value { get; set; }
```

Comment:
tsk-tsk copy-paste :-)
maybe introduce abstract NonZero<> to avoid such and add a common check for 0